administration/command-line-interface.md

```diff
@@ -34,62 +35,132 @@ docker-compose run --rm cli azuracast_cli [command]
 php /var/azuracast/www/bin/console [command]
 ```
 
-# Command Reference
+# Account Related CLI Commands
+This section will cover all account related CLI commands and the response the CLI will provide.  
 
-<br>
+## List all accounts
+```
+(cli_command) azuracast:account:list 
 
-## List All Commands
+AzuraCast User Accounts
+=======================
 
-To see all commands available, run the appropriate CLI command for your installation type with no command specified.
+ ---------------- ------ --------------------- -------------------
+  E-mail Address   Name   Roles                 Created
+ ---------------- ------ --------------------- -------------------
+  name@domain.com         Super Administrator   YYYY-MM-DD HH:MM
+ ---------------- ------ --------------------- -------------------
+```
+message here
 
-<br>
+## Create a unique login recovery URL for the specified account.
+``` 
+(cli_command) azuracast:account:login-token (EMAIL)
 
-## Reset Administrator Password
+Generate Account Login Recovery URL
+===================================
 
-```bash
-(cli_command) azuracast:account:reset-password your@email.com
+ The account recovery URL is:
+
+     https://domain/recover/hash:hash
+
+ Log in using this temporary URL and set a new password using the web interface.
 ```
+message here
```

Comment:
Are those `message here` parts intended to be there?

Reply:
Whoops, forgot about them strings, all removed.

I was going to add notes detailing what exactly the commands are, but they're self explanatory.